app/src/main/java/org/wikipedia/analytics/metricsplatform/MetricsPlatform.kt

```diff
@@ -19,5 +20,9 @@ object MetricsPlatform {
         WikipediaApp.instance.wikiSite.authority()
     )
 
-    val client: MetricsClient = MetricsClient.builder(clientData).build()
+    val client: MetricsClient = MetricsClient.builder(clientData)
+        .eventQueueCapacity(256)
```

Comment:
@dbrant 
Can we keep the queue size a bit lower? 256 is almost an average session's worth of user interactions, in case we drop it for the edge case reasons. May be stick to our previous 128?

Reply:
How about like this?

Reply:
Even better! 🎉